scripts/build.py

```diff
@@ -219,6 +219,11 @@ def modify_constants(prod_env, maintenance_mode):
         enable_maintenance_mode_variable)
 
 
+def set_constants_to_default():
+    """Set variables in constants.ts and feconf.py to default values."""
+    modify_constants(prod_env=False, maintenance_mode=False)
```

Comment:
Do we need prod_env here?

Reply:
No, but it simplifies the reading of this, since `modify_constants(False, maintenance_mode=False)` is less readable because the developer needs to look at what the first argument means.

Reply:
I agree this looks good now (I think earlier the first one was not a keyword argument!)